src/graph/GoExecutor.cpp

```diff
@@ -1315,28 +1315,24 @@ bool GoExecutor::processFinalResult(Callback cb) const {
     return true;
 }
 
-OptVariantType GoExecutor::VertexHolder::getDefaultProp(TagID tid, const std::string &prop) const {
-    for (auto it = data_.cbegin(); it != data_.cend(); ++it) {
-        auto it2 = it->second.find(tid);
-        if (it2 != it->second.cend()) {
-            return RowReader::getDefaultProp(std::get<0>(it2->second).get(), prop);
-        }
+OptVariantType GoExecutor::VertexHolder::getDefaultProp(
```

Comment:
Actually, we don't need "getDefaultProp" and "getDefaultPropType" inside VertexHolder, 

You could use RowReader::getDefaultProp directly.